tests/unit/modules/network/nxos/test_nxos_snmp_server.py

```diff
@@ -294,6 +294,31 @@ def test_nxos_snmp_server_linear_overridden(self):
         result = self.execute_module(changed=True)
         self.assertEqual(set(result["commands"]), set(commands))
 
+    def test_nxos_snmp_server_location_spaces(self):
+        # test replaced for linear attributes
+        self.get_config.return_value = dedent(
+            """\
+            snmp-server contact test-switch @t localhost
+            snmp-server location long/and.(complicated) address
+            """
+        )
+        set_module_args(
+            dict(
+                config=dict(
+                    contact="test-switch @t localhost",
+                    location="lab",
+                ),
+                state="replaced",
+            ),
+            ignore_provider_arg,
+        )
+        commands = [
```

Comment:
@tin-ot CI is currently complaining about this unit test case failing. This is because, `snmp-server contact test-switch @t localhost` is already defined in the `get_config.return_value`, which means that the task execution would result in only `snmp-server location lab` being sent.

If the goal is to verify that facts gathering is working fine with contact and location values having whitespace in them, this has to be an idempotent test case.
```suggestion
        commands = []
```